pkg/cortex/modules.go

```diff
@@ -366,6 +382,22 @@ func (t *Cortex) initIngester(cfg *Config) (serv services.Service, err error) {
 	return t.ingester, nil
 }
 
+func (t *Cortex) initFlusher(cfg *Config) (serv services.Service, err error) {
+	t.flusher, err = flusher.New(
+		cfg.Flusher,
+		cfg.Ingester,
+		cfg.IngesterClient,
+		t.store,
+		prometheus.DefaultRegisterer,
+	)
+	if err != nil {
+		return
+	}
+
+	t.server.HTTP.Path("/ready").Handler(http.HandlerFunc(t.flusher.ReadinessHandler))
```

Comment:
We already set global /ready handler at cortex.Run. Can you move flusher readiness handler there as well? Similar to how ingester's readiness handler is hooked.

Reply:
Didn't notice that change with services. Will remove that.